javalib-ext-dummies/src/main/scala/java/security/SecureRandom.scala

```diff
@@ -0,0 +1,19 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package java.security
+
+/** Fake implementation of `SecureRandom` that is not actually secure at all.
+ *
+ *  It directly delegates to `java.util.Random`.
+ */
+class SecureRandom() extends java.util.Random
```

Comment:
Are the parens here javalib style? I haven't seen this before.

E.g. InputStream (admittedly old) doesn't have this.

https://github.com/scala-js/scala-js/blob/058532aa8c504b76431b40e3e1b51b2cdef87643/javalib/src/main/scala/java/io/InputStream.scala#L15

Reply:
I removed the `()`.